script.js

```diff
@@ -0,0 +1,61 @@
+let buttonCalculate = document.querySelector(".btn");
+buttonCalculate.addEventListener('click', BMICalculate);
+
+function BMICalculate() {
+    let height = parseInt(document.querySelector(".height").value);
+    let weight = parseInt(document.querySelector(".weight").value);
+    let resultValue = document.querySelector(".result")
+    if (height === "" || isNaN(height)) {
+       // alert("please Enter the Height");
+         resultValue.innerHTML="please enter the height"
```

Comment:
[Docs for innerHTML](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML) recommend that instead of innerHTML you use [textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent) when inserting plain text.

```suggestion
         resultValue.textContent = "please enter the height"
```